Sources/Hyperwallet.swift

```diff
@@ -573,8 +573,6 @@ public final class Hyperwallet {
     /// `HyperwalletReceiptQueryParam` object, if it is not nil. Otherwise the default ordering and
     /// filtering will be applied.
     ///
-    /// * Offset: 0
```

Comment:
Update the comment to:
' /// Returns the list of receipts for the User associated with the Prepaid card token.
   ///
   /// The filtering of `HyperwalletReceipt` will be based on the criteria specified within the
   /// `HyperwalletReceiptQueryParam` object. CreatedAfter needs to be provided to get the receipts.
  /// Receipts are returned sorted in ascending order of creation date
   ///
   /// * Created Before: N/A
   /// * Created After: “Some Date”
   /// * Currency: All '

Reply:
Updated